src/test/java/org/broadinstitute/hellbender/tools/spark/sv/NovelAdjacencyReferenceLocationsUnitTest.java

```diff
@@ -449,30 +462,66 @@ public void testGetBreakpoints_tandemDuplication_complex() throws IOException {
         Assert.assertEquals(breakpoints.leftJustifiedLeftRefLoc, new SimpleInterval("20", 312609, 312609));
         Assert.assertEquals(breakpoints.leftJustifiedRightRefLoc, new SimpleInterval("20", 312609, 312609));
         Assert.assertEquals(breakpoints.endConnectionType, FIVE_TO_THREE);
     }
 
-    //     // commenting out for now because it is a simple translocation
-//    @Test
-//    public void testGetAssembledBreakpointsFromAlignmentRegions() throws Exception {
-//        final byte[] contigSequence = "GACGAACGATTTGACTTTAATATGAAATGTTTTATGTGGGCTATAAAATTATCCAAACTCGACACAGGACATTTTGAGCTTATTTCCAAATCATCTGGCCTTCATCTACCCACTGGAACTATTACTCTGCTGGGTCCTCATGGAAACATATCTTTCAGCCCTAACAATGAGACTACAGACATCTACGTCCCCAACACAACAGCTAAAAAGCAGTAGAATGTCAGAAAGGCTATCCACTTAGCCCTTGGCTGACAGGCCCCACTGAGCATCCTTTGCGAAGTCCATTTACTAGCTAATTCATAATTTACACAAGGCATTCAGACATAGCAGCTAAGATATAAAACATTTATCAACACAGGGACTAGTTTGTCATTTTAAAATAATTATGTTTAAGTAAGCCAATAAAGTCTATCTTCTCCAATTTACTTATTGAGCTTTATGAGGCAATTTAAGTCCCGATTTTGGGGGGTATGTATGAAAGGAGAGCATGGAAATGCCATTTGCTCCCTGAAGTTTTTATCTTTTTTTTTTTGAGATAGAGTCTTGTGTTTTCTGTGGAGTACATGAGTATGCATCAAAGCTAACAACGCCCACTGCCCTGTTAGTCAAATACCTTTGA".getBytes();
-//        final AlignmentRegion region1 = new AlignmentRegion("1", "contig-1", new SimpleInterval("21", 118873207, 118873739), TextCigarCodec.decode("532M87S"), true, 60, 0, 1, 532);
-//        final AlignmentRegion region2 = new AlignmentRegion("1", "contig-1", new SimpleInterval("20", 175705642, 175705671), TextCigarCodec.decode("518S29M72S"), false, 3, 0, 519, 547);
-//        final AlignmentRegion region3 = new AlignmentRegion("1", "contig-1", new SimpleInterval("20", 118875262, 118875338), TextCigarCodec.decode("543S76M"), false, 60, 0, 544, 619);
-//        final List<AlignmentRegion> alignmentRegionList = Arrays.asList(region1, region2, region3);
-//        final List<ChimericAlignment> assembledBreakpointsFromAlignmentRegions = ChimericAlignment.fromSplitAlignments(new Tuple2<>(alignmentRegionList, contigSequence), SVCallerTestDataProvider.seqDict);
-//        Assert.assertEquals(assembledBreakpointsFromAlignmentRegions.size(), 1);
-//        final ChimericAlignment chimericAlignment = assembledBreakpointsFromAlignmentRegions.get(0);
-//        Assert.assertEquals(chimericAlignment.regionWithLowerCoordOnContig.contigId, "contig-1");
-//        Assert.assertEquals(chimericAlignment.regionWithLowerCoordOnContig, region1);
-//        Assert.assertEquals(chimericAlignment.regionWithHigherCoordOnContig, region3);
-//        final NovelAdjacencyReferenceLocations breakpoints = new NovelAdjacencyReferenceLocations(chimericAlignment);
-//        Assert.assertTrue(breakpoints.complication.homologyForwardStrandRep.isEmpty());
-//        Assert.assertEquals(breakpoints.complication.insertedSequenceForwardStrandRep, "GAGATAGAGTC");
-//    }
+    // TODO: 4/3/17 enable when transolocations can be handled
+    @Test(enabled = false)
+    public void testGetAssembledBreakpointsFromAlignmentRegions() throws Exception {
```

Comment:
Does this test the breaking use case? Could you add a comment or two to the test describing what it's testing?